filplus.md

```diff
@@ -0,0 +1,100 @@
+# Filplus Retrieval Sampling
+
+## Background
+
+The goal of filplus retrieval sampling is to offer an aggregated view of how each storage provider performs in terms of
+retrieval.
+It is a sampling of the retrieval performance of the storage providers in the Filecoin network.
+
+## Sampling Logic
+
+All active verified deals in Filecoin network will be sampled randomly.
+
+Newer deal will have a higher chance to be sampled than older deals. The chance is 4x for each year the deal is
```

Comment:
Newer deal -> Newer deals
The chance is 4x for each year the deal is newer -> The chance is 4x for each year the deal is newer determined by deal transaction date.